src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
Looks good, but don't slice variable's name.

Reply:
Agree with Igor, name it `calculator` or something like this